google-cloud-clients/google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/EnhancedBigtableStub.java

```diff
@@ -75,6 +79,8 @@
 public class EnhancedBigtableStub implements AutoCloseable {
   private static final String TRACING_OUTER_CLIENT_NAME = "Bigtable";
   private static final String TRACING_INNER_CLIENT_NAME = "BaseBigtable";
+  private static final MutateRowsBatchingDescriptorV2 batchingDescriptorV2 =
```

Comment:
is this going to live side by side with MutateRowsBatchingDescriptor? Can we replace the old one with this?

Reply:
Thanks for the review.
Yes, We are aiming to remove existing `MutateRowsBatchingDescriptor`. I have added a TODO in `MutateRowsBatchingDescriptorV2` for the same reason.